chess/__init__.py

```diff
@@ -626,14 +626,15 @@ def __init__(self, from_square, to_square, promotion=None, drop=None):
         self.promotion = promotion
         self.drop = drop
 
-    def uci(self):
+    def pacn(self):
         """
-        Gets an UCI string for the move.
+        Gets an PACN string for the move.
```

Comment:
What does `PACN` stand for? I couldn't find the term in the XBoard protocol spec (or anywhere else).